src/benchmarks/real-world/Microsoft.ML.Benchmarks/Microsoft.ML.Benchmarks/Harness/Metrics.cs

```diff
@@ -0,0 +1,110 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Columns;
+using BenchmarkDotNet.Reports;
+using BenchmarkDotNet.Running;
+
+namespace Microsoft.ML.Benchmarks
+{
+    public abstract class WithExtraMetrics
```

Comment:
this feature is not used in the benchmarks that you have ported (none of the  classes derive from it), it can be removed